lib/msf/core/payload/python/meterpreter_loader.rb

```diff
@@ -358,6 +359,13 @@ def dec_b(self,ct):
 ?
   end
 
+  def valid_path?(path)
```

Comment:
Random thoughts:
- This name is a bit of a lie since it's validating `rpath` rather than just generic paths 👀 
- The other meterpreters will want similar parsing constructs, we may need to move this somewhere generic so it can be reused
- Do we provide any useful feedback to the user if they provide an invalid value without realizing, i.e. if they specified `path:/tmp/foo.txt` would this silently do nothing? 👀 
- This matches `rpath:`, I don't know if that's problematic or not:
```
3.1.1 :008 > "rpath:".match?(/^rpath:(\w?\p{Punct}?)/)
 => true 
```
- Since the regex isn't anchored, and it matches successfully again just `rpath:`, it also accepts extra junk being present:
```
3.1.1 :013 > "rpath:😊😊😊".match?(/^rpath:(\w?\p{Punct}?)/)
 => true 
```
- Potentially consider introducing a generic parse method which returns a hash with `{ rpath: "..." }` instead of having a separate validate method, and parsing it yourself on line 77 - that way the implementation can be easily extended in the future, and extracted to a centralized location for other meterpreters to use. This pattern will also allow it to be more easily unit tested too

Reply:
Single quotes would also break things, as we'd breakout of the python string. But that scenario is definitely pebkac
```
3.1.1 :014 > "rpath:'".match?(/^rpath:(\w?\p{Punct}?)/)
 => true 
```